lib/xdr/contract/sc_static.ex

```diff
@@ -0,0 +1,55 @@
+defmodule StellarBase.XDR.SCStatic do
+  @moduledoc """
+  Representation of Stellar `SCStatic` type.
+  """
+  @behaviour XDR.Declaration
```

Comment:
Add new line between moduledoc and `@behaviour`. Updates in all modules where applicable!